### ISSUE TITLE
Crash when viewing recent wiki updates

### ISSUE THREAD

**Issue by yan12125** [slapperwan/gh4a#843]
Octodroid version: v4.3.1 from F-Droid. Please tell me if you want me to test against git.

Steps to reproduce:
1. Navigate to https://github.com/varjolintu/keepassxc-browser
2. Hit "Recent Wiki Updates"

Stack trace:
```
--------- beginning of crash04-10 00:07:25.309  27658    27658         AndroidRuntime  E  FATAL EXCEPTION: main
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  Process: com.gh4a, PID: 27658
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  io.reactivex.exceptions.CompositeException: 1 exceptions occurred.
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  at io.reactivex.internal.observers.ConsumerSingleObserver.onError(ConsumerSingleObserver.java:47)
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  at com.philosophicalhacker.lib.ReactiveLoaders$LoaderSingle$1.onError(ReactiveLoaders.java:51)
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  at com.philosophicalhacker.lib.RxLoaderCallbacks.onLoadFinished(RxLoaderCallbacks.java:33)
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  at android.support.v4.app.LoaderManagerImpl$LoaderInfo.callOnLoadFinished(LoaderManager.java:476)
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  at android.support.v4.app.LoaderManagerImpl$LoaderInfo.onLoadComplete(LoaderManager.java:444)
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  at android.support.v4.content.Loader.deliverResult(Loader.java:126)
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  at com.philosophicalhacker.lib.RxLoaderCallbacks$RxAndroidLoader.safeDeliverResult(RxLoaderCallbacks.java:109)
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  at com.philosophicalhacker.lib.RxLoaderCallbacks$RxAndroidLoader.access$000(RxLoaderCallbacks.java:44)
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  at com.philosophicalhacker.lib.RxLoaderCallbacks$RxAndroidLoader$2.accept(RxLoaderCallbacks.java:85)
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  at com.philosophicalhacker.lib.RxLoaderCallbacks$RxAndroidLoader$2.accept(RxLoaderCallbacks.java:82)
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  at io.reactivex.internal.observers.ConsumerSingleObserver.onError(ConsumerSingleObserver.java:44)
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  at io.reactivex.internal.operators.single.SingleObserveOn$ObserveOnSingleObserver.run(SingleObserveOn.java:79)
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  at io.reactivex.android.schedulers.HandlerScheduler$ScheduledRunnable.run(HandlerScheduler.java:109)
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  at android.os.Handler.handleCallback(Handler.java:739)
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  at android.os.Handler.dispatchMessage(Handler.java:95)
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  at android.os.Looper.loop(Looper.java:148)
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  at android.app.ActivityThread.main(ActivityThread.java:5451)
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  at java.lang.reflect.Method.invoke(Native Method)
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  ComposedException 1 :
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  java.lang.RuntimeException: java.lang.NoSuchMethodException: <init> [interface org.simpleframework.xml.core.Contact, interface org.simpleframework.xml.ElementList, class org.simpleframework.xml.stream.Format]
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  at retrofit2.converter.simplexml.SimpleXmlResponseBodyConverter.convert(SimpleXmlResponseBodyConverter.java:44)
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  at retrofit2.converter.simplexml.SimpleXmlResponseBodyConverter.convert(SimpleXmlResponseBodyConverter.java:23)
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  at retrofit2.ServiceMethod.toResponse(ServiceMethod.java:118)
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  at retrofit2.OkHttpCall.parseResponse(OkHttpCall.java:212)
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  at retrofit2.OkHttpCall.execute(OkHttpCall.java:174)
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  at retrofit2.adapter.rxjava2.CallExecuteObservable.subscribeActual(CallExecuteObservable.java:41)
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  at io.reactivex.Observable.subscribe(Observable.java:10700)
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  at io.reactivex.internal.operators.observable.ObservableSingleSingle.subscribeActual(ObservableSingleSingle.java:35)
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  at io.reactivex.Single.subscribe(Single.java:2692)
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  at io.reactivex.internal.operators.single.SingleMap.subscribeActual(SingleMap.java:33)
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  at io.reactivex.Single.subscribe(Single.java:2692)
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  at io.reactivex.internal.operators.single.SingleMap.subscribeActual(SingleMap.java:33)
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  at io.reactivex.Single.subscribe(Single.java:2692)
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  at io.reactivex.internal.operators.single.SingleResumeNext.subscribeActual(SingleResumeNext.java:39)
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  at io.reactivex.Single.subscribe(Single.java:2692)
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  at io.reactivex.internal.operators.single.SingleResumeNext.subscribeActual(SingleResumeNext.java:39)
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  at io.reactivex.Single.subscribe(Single.java:2692)
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  at io.reactivex.internal.operators.single.SingleSubscribeOn$SubscribeOnObserver.run(SingleSubscribeOn.java:89)
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  at io.reactivex.Scheduler$1.run(Scheduler.java:138)
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  at io.reactivex.internal.schedulers.ScheduledRunnable.run(ScheduledRunnable.java:59)
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  at io.reactivex.internal.schedulers.ScheduledRunnable.call(ScheduledRunnable.java:51)
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  at java.util.concurrent.FutureTask.run(FutureTask.java:237)
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:154)
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:269)
04-10 00:07:25.309  27658    27658         AndroidRuntime  E  at java.util.concurre
04-10 00:07:25.312   1778     3469        ActivityManager  D  Prepare to send crash information Messege to SystemGaService
04-10 00:07:25.313   1778     3469        ActivityManager  W  Force finishing activity com.gh4a/.activities.WikiListActivity
```

**Comment by maniac103** [comment]
This was fixed already in #838 :-)

**Comment by yan12125** [comment]
Oh, thanks! I build git-master and it runs fine! Looking forward to a new release.
